Enterprise-CMCS/macpro-onemac
# Fix broken styles for waiver attachments box

### Details

This is a quick fix for some broken styles introduced in the Waiver Attachments box in the Detail page

### Changes

Broken
<img width="1120" alt="Screen Shot 2023-04-27 at 11 49 56 AM" src="https://user-images.githubusercontent.com/4022304/234917667-cdbd3312-8505-41f8-b415-7daafdd7b656.png">

Fixed
<img width="1077" alt="Screen Shot 2023-04-27 at 11 49 19 AM" src="https://user-images.githubusercontent.com/4022304/234917693-ab39c46d-3773-46e2-981b-879b1ada1b74.png">


### Implementation Notes

- Developer focused info that may affect future work

### Test Plan

1. Ensure styles on Waiver Details page are as intended
2. Ensure styles on New Submission cards is as intended from the previous PR, still. #1225
